src/System.Management.Automation/engine/MshCmdlet.cs

```diff
@@ -83,7 +83,7 @@ public bool IsPresent
         /// </summary>
         /// <param name="switchParameter">The SwitchParameter object to convert to bool.</param>
         /// <returns>The corresponding boolean value.</returns>
-        public static implicit operator bool (SwitchParameter switchParameter)
+        public static implicit operator bool(SwitchParameter switchParameter)
```

Comment:
Is removing the whitespace correct here?  seems to make it harder to read

Reply:
Yes, it is correct. Using predefined `bool` confuses. If we'd use another name it would look like method definition.